ci-operator/config/redhat-appstudio/jvm-build-service/redhat-appstudio-jvm-build-service-main.yaml

```diff
@@ -6,9 +6,11 @@ build_root:
 images:
 - dockerfile_path: Dockerfile
   to: redhat-appstudio-jvm-build-service-image
-- dockerfile_path: Dockerfile.cache-in-ci
+- context_dir: java-components
+  dockerfile_path: cache/src/main/docker/Dockerfile.all-in-one
   to: redhat-appstudio-jvm-build-service-cache-image
-- dockerfile_path: Dockerfile.reqprocessor-in-ci
+- context_dir: java-components
+  dockerfile_path: build-request-processor/src/main/docker/Dockerfile.reqprocessor-in-ci
```

Comment:
How did that pass? This file does not exist, it is `Dockerfile.all-in-one`.

Reply:
you've got to manually run the the rehearsals

the test in question did not run after I pushed this update, and I forgot to trigger the test before my EOB :-) ... that is why they are all green

let me fix, the re-run rehearsals, and then will highlight the job

that said, I see you made the correct change in you PR ... I'll run the rehearsals there, and tag for merge